packages/embark/src/lib/modules/solidity/solcP.js

```diff
@@ -80,6 +82,10 @@ class SolcProcess extends ProcessWrapper {
 let solcProcess;
 process.on('message', (msg) => {
   if (msg.action === "init") {
+    if (msg.options.logger) {
+      const loggerOptions = {...msg.options.logger, events: new Events() };
+      msg.options.logger = new Logger(loggerOptions);
```

Comment:
Don't we have this: https://github.com/embark-framework/embark/blob/master/packages/embark/src/lib/core/processes/processWrapper.js#L50-L61
To do exactly that?

Reply:
Like Anthony said. Also, you can just pass `console` as a logger and it will be intercepted

Reply:
Ah, this might be what @jrainville was referring to on Gitter.

Lemme give this a shot